test/unit/event/send-reset-event.test.js

```diff
@@ -17,18 +17,18 @@ jest.mock('ffc-pay-event-publisher', () => {
   }
 })
 jest.mock('../../../app/config')
-const config = require('../../../app/config')
+const { processingConfig, messageConfig } = require('../../../app/config')
 const { PAYMENT_RESET } = require('../../../app/constants/events')
 const { SOURCE } = require('../../../app/constants/source')
-const sendResetEvent = require('../../../app/event/send-reset-event')
+const { sendResetEvent } = require('../../../app/event/send-reset-event')
 
 let paymentRequest
 
 beforeEach(() => {
-  paymentRequest = JSON.parse(JSON.stringify(require('../../mocks/payment-request')))
+  paymentRequest = JSON.parse(JSON.stringify(require('../../mocks/payment-requests/payment-request')))
 
-  config.useV2Events = true
-  config.eventsTopic = 'v2-events'
+  processingConfig.useV2Events = true
+  messageConfig.eventsTopic = 'v2-events'
 })
```

Comment:
Should the before and after each not be defined in the describe?

Reply:
Response as before, this allows them to be shared for each describe :)